### PR TITLE
Add Sentry capabilities to Cromwell. Closes #2134

### LINKED ISSUE #2134: Add Sentry magic to CromIAM
Might be worth talking to @ansingh7115

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

File: core/src/main/resources/logback.xml [hunk 1]
```diff
@@ -77,11 +77,20 @@
     </else>
   </if>
 
+  <!-- Configure the Sentry appender, overriding the logging threshold to the WARN level -->
+  <appender name="Sentry" class="com.getsentry.raven.logback.SentryAppender">
```
Comment (cjllanwarne) [inline core/src/main/resources/logback.xml]: Does this do anything for Mr & Mrs Average User running Cromwell on their laptops?

**Comment by cjllanwarne** [comment]
Can we have some usage docs? Or is it genuinely just "magic"?

File: core/src/main/resources/logback.xml (hunk 1 above)
Comment (geoffjentry) [inline core/src/main/resources/logback.xml]: Nope

**Comment by geoffjentry** [comment]
@cjllanwarne we could, but it's really just for our internal deployments with devops so imo it's not necessary and would just add to the visual noise of our docs. It'd only take a couple of minutes to create said visual noise if you really think it'd be helpful

**Comment by Horneth** [comment]
Have you verified that it's doing something good or do we just trust it ? Otherwise 👍

**Comment by geoffjentry** [comment]
@Horneth Indeed I have. I managed to set off alarms on the firecloud dev sentry project ;)

**Comment by cjllanwarne** [comment]
👍

File: README.md [hunk 2]
```diff
@@ -1873,6 +1873,8 @@ workflow-options {
 
 The usual case of generating the temporary per workflow logs is to copy them to a remote directory, while deleting the local copy to preserve local disk space. To specify the remote directory to copy the logs to use the separate [workflow option](#workflow-options) `final_workflow_log_dir`.
 
+Cromwell supports [Sentry](https://docs.sentry.io), a service that can be used to monitor exceptions happening in an application. To make use of this add `-Dsentry.dsn=DSN_URL` with your DSN URL to your Java command line.
```
Comment (cjllanwarne) [inline README.md]: 💯

**Pull request merged**